preprocessing/text_data/04_preprocessing_text_data_v3.ipynb

```diff
@@ -0,0 +1,1955 @@
+{
```

Comment:
<p>Add space:</p><blockquote>task(e.g. sentiment analysis)</blockquote><p> </p><p>Suggest updating (for readability and style):</p><blockquote>Here for twitter data, numbers are not super important in our case, so we will remove those.</blockquote><p>To something like</p><blockquote>In this example, we are using twitter data (tweets) and typically, numbers are not that important for understanding the meaning or content of a tweet. Therefore, we will remove numbers.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>